client/src/app/shared/custom-button/custom-button.component.ts

```diff
@@ -10,5 +11,18 @@ export class CustomButtonComponent {
   @Input() icon: string = '';
   @Input() type: string = '';
 
-  constructor() {}
+  handleClick(): void {
+    if (this.type === 'create' && this.text === 'Add Account') {
```

Comment:
if (this.type === 'create') {
    if (this.text === 'Add Account') {}
    if (this.text === 'Add Transaction') {}
....
}